### PR TITLE
Upgrade to Pulumi v1.12.1 and new schema generator

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: examples/aws_elastigroup/csharp/ElastiGroupStack.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        var sg = new Pulumi.Aws.Ec2.SecurityGroup("my-csharp-security-group");
+
+        var azs = Invokes.GetAvailabilityZones();
+        
+        var elastigroup = new Elastigroup("my-csharp-elastigroup", new ElastigroupArgs
+        {
+            FallbackToOndemand = false,
+            InstanceTypesOndemand = "m3.2xlarge",
+            InstanceTypesSpots = new InputList<string>
```
Comment (mikhailshilkov) [inline examples/aws_elastigroup/csharp/ElastiGroupStack.cs]:
```suggestion
            InstanceTypesSpots =
```

File: examples/aws_elastigroup/csharp/ElastiGroupStack.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            {
+                "m3.xlarge",
+                "m3.2xlarge",
+            },
+            Orientation = "balanced",
+            Product = "Linux/UNIX",
+            SecurityGroups = new InputList<string>
```
Comment (mikhailshilkov) [inline examples/aws_elastigroup/csharp/ElastiGroupStack.cs]:
```suggestion
            SecurityGroups =
```

File: examples/examples_test.go [hunk 3]
```diff
@@ -64,3 +85,31 @@ func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
...
+			"Pulumi.SpotInst",
+		},
+	})
+
+	return baseJS
+}
+
+func getPythonBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	base := getBaseOptions()
+	baseJS := base.With(integration.ProgramTestOptions{
```
Comment (mikhailshilkov) [inline examples/examples_test.go]: Same nit as in the other provider

File: examples/aws_elastigroup/csharp/ElastiGroupStack.cs (hunk 1 above)
Comment (stack72) [reply examples/aws_elastigroup/csharp/ElastiGroupStack.cs]: Fixed :)

File: examples/aws_elastigroup/csharp/ElastiGroupStack.cs (hunk 2 above)
Comment (stack72) [reply examples/aws_elastigroup/csharp/ElastiGroupStack.cs]: Fixed :)

File: examples/examples_test.go (hunk 3 above)
Comment (stack72) [reply examples/examples_test.go]: Fixed :)

**Pull request merged**